main/templates/main/index.html

```diff
@@ -3,78 +3,59 @@
 {% block content %}
 
   <div class="container">
-    <div class="jumbotron">
-      <h1>Juno's <i>Personal Data Exploratory</i></h1>
-      <p>
-        📊📓 Personal Data Notebooks allow you to explore and analyze your personal data that is stored in <i>Open Humans</i>
-        right in your web browser. The <i>Exploratory</i> allows you to find and use interesting notebooks right away.
-      </p>
-  </div>
     <div class="row">
-      <div class="col-md-6">
-        <h2>Data Sources</h2>
+      <div class="col-xs-12">
         <p>
-          The <i>Exploratory</i> contains ready-to-use notebooks for many different data sources which are on Open Humans.
-          Click the buttons below to find notebooks that make use of the different data sources:
+          <b>Want to explore your data in Open Humans?
+          This gallery will help.</b>
+          There are a bunch of notebooks that you can use.
+          See if you find one you'd like to try – please give it
+          a ♥ if you like it!
         </p>
-        <p>
-        {%for source in data_sources%}
-          <a href="{%url 'data-sources' source%}" class="btn btn-primary btn-primary-hollow btn-lg padded">
-          {{source}}
-        </a>
-        {%endfor%}
-        </p>
-        <a
-          {% if client_id %}
-          class="btn btn-primary btn-lg"
-          href="https://www.openhumans.org/direct-sharing/projects/oauth2/authorize/?client_id={{ client_id }}&response_type=code&redirect_uri={{ redirect_uri }}"
-          {% else %}
-          class="btn btn-default disabled"
-          {% endif %}
-          >
-          Get started
+        <hr>
+        {% include 'main/partials/notebooks-table.html' %}
```

Comment:
The filtering is nice, but I fear it will not scale with a growing number of data sources as you'll end up with tons of buttons above every table. The same problem already appears now on smaller screens.

Reply:
Yes, the current version has the same issue as the view is providing an infinite list. Two solutions (1) limit the list (2) don't use this table display on xs screens (provide an alternate view). (1) seems straightforward; do you have any feelings on whether (2) should be done?